store/index.js

```diff
@@ -627,6 +627,20 @@ export const getters = {
         return toolGroup;
     },
 
+    getOriginalColumnValue: (p_state) => (p_subjectID, p_columnName) => {
+
+        for ( let index = 0; index < p_state.dataTable.original.length; index++ ) {
+
+            if ( p_subjectID === p_state.dataTable.original[index]["participant_id"] ) {
+
+                return p_state.dataTable.original[index][p_columnName];
+            }
+        }
+
+        return null;
```

Comment:
More out of curiosity: is there any happy path that should lead to this return?
If not, what do you want to happen if an unhappy path leads here? 

I'm assuming this would raise some JS equivalent of a keyError in the `transformedValue` method
of the component? That might quite confusing to debug, so maybe catching it here would be easier.